app/viewmodels/checkAnswers/sections/journeyType/HowMovementTransportedSummary.scala

```diff
@@ -50,7 +52,13 @@ object HowMovementTransportedSummary {
               ).url,
               id = HowMovementTransportedPage
             ).withVisuallyHiddenText(messages("howMovementTransported.change.hidden"))
-          )} else Seq()
+          )
         )
     }
+
+  private def showChangeLink(implicit request: DataRequest[_]): Boolean =
```

Comment:
Would it be clearer to remove the negation in this method, and at the call site, and just call this something like `hideChangeLink` instead?

Reply:
Done in [bdaf620](https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/30/commits/bdaf62069222611d2f5a35a5907fc3aad754e370)